launch_gz/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model
+  href="http://download.ros.org/schema/package_format2.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>launch_gz</name>
```

Comment:
Should we name this to `ros_gz_launch` or something similar to align with the rest of the packages in this repo? For example, the deb package for this would something like `ros-jazzy-launch-gz` whereas all the other packages in this repo are `ros-jazzy-ros-gz-<pkg>`.

Reply:
Do we even need this to be it's own package? Can it not live in `ros_gz_sim`?

Reply:
I ended up moving it to `ros_gz_sim` in [01345e2](https://github.com/gazebosim/ros_gz/pull/528/commits/01345e2d95d76998f3ab8fbd341e2164d075a009).